### PR TITLE
removes atmos and bombs for lag

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
Screenshot_2577
personal request

initial testing

https://user-images.githubusercontent.com/15794172/211003243-687bc5ac-d231-4850-8422-b3c458a3542e.mp4

## Why It's Good For The Game
causes lag
## Changelog
:cl:
performance: the game will no longer need to simulate atmospherics or ttv's exploding
/:cl:

### CODE REVIEW

**Comment by Kylerace** [comment]
this might cause issues with breathing but anything to remove lag

**Comment by Serxule** [comment]
i ageee agree i agree good pr merge now

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 1]
```diff
@@ -55,13 +55,13 @@
 	#endif
```
Comment (Kylerace) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm:56]:
```suggestion
/datum/gas_mixture/immutable/space/normal
	initial_temperature = ROOM_TEMP
```

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 2]
```diff
@@ -55,13 +55,13 @@
...
 
 /turf/open/Initialize(mapload)
-	if(!blocks_air)
-		air = create_gas_mixture()
+	/*if(!blocks_air)
+		//air = create_gas_mixture()
```
Comment (Kylerace) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm:59]:
```suggestion
	if(!blocks_air)
		air = new/datum/gas_mixture/immutable/space/normal /*
```

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 3]
```diff
@@ -53,15 +53,16 @@
 	#ifdef TRACK_MAX_SHARE
 	var/max_share = 0
 	#endif
-
+/datum/gas_mixture/immutable/space/normal
+	initial_temperature = ROOM_TEMP
```
Comment (Kylerace) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm:57]:
```suggestion
	initial_temperature = 293
```

**Comment by SyncIt21** [comment]
Clicking dead people is much faster. They don't move around and so don't cause any lag. Approved 👍

**Comment by CRITAWAKETS** [comment]
lungs are now obsolete, you should probably remove them to reduce unused objects and therefore lag.

**Comment by Pinkufeck** [comment]
You only removed oxygen not all the gases, misleading PR.

**Pull request closed without merging**